api/api.go

```diff
@@ -76,6 +76,14 @@ func (srv *Server) initAPI(addr string) {
 		handleHTTPRequest(mux, "/miner/submitblock", srv.minerSubmitblockHandler)
 		handleHTTPRequest(mux, "/miner/headerforwork", srv.minerHeaderforworkHandler)
 		handleHTTPRequest(mux, "/miner/submitheader", srv.minerSubmitheaderHandler)
+		handleHTTPRequest(mux, "/miner/connecttopool", srv.minerConnectToPoolHandler)
+		handleHTTPRequest(mux, "/miner/poolheaderforwork", srv.minerPoolHeaderForWorkHandler)
+		handleHTTPRequest(mux, "/miner/submitpoolheader", srv.minerSubmitPoolHeaderHandler)
```

Comment:
to match the existing API calls, I would suggest `/miner/pool/connect`, `/miner/pool/headerforwork`, and `/miner/pool/submitheader`